src/app/state/transaction/types.ts

```diff
@@ -32,6 +37,10 @@ export interface Transaction {
   timestamp: number | undefined
   to: string | undefined
   type: TransactionType
+  method: string | undefined
```

Comment:
These seem like they should be consolidated

Reply:
yep, new prop was redundant. done